<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Vm_FindGoods</class>
 <widget class="QWidget" name="Vm_FindGoods">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lb_tyoe">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>30</y>
     <width>39</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>类型:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>501</width>
     <height>171</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_brand">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>70</y>
     <width>39</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>品牌:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cob_brand">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QComboBox" name="cob_type">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_commit_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_commit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
